browser/extensions/BUILD.gn

```diff
@@ -219,24 +191,16 @@ source_set("extensions") {
     deps += [ "//brave/components/web_discovery/common" ]
   }
 
-  # It seems like this brave_wallet_api should be renamed to ethereum_remote_client_api.
-  # However this is not possible right now because the ethereum-remote-client extension
-  # uses chrome.braveWallet, so the API is intentionally not being renamed now.
-  if (ethereum_remote_client_enabled) {
-    sources += [
-      "api/brave_wallet_api.cc",
-      "api/brave_wallet_api.h",
-    ]
-
-    deps += [
-      ":ethereum_remote_client_utils",
-      "//brave/browser/ethereum_remote_client",
-      "//brave/browser/ethereum_remote_client:browser",
-      "//brave/components/brave_wallet/browser",
-      "//brave/components/brave_wallet/common:buildflags",
-      "//components/prefs",
-    ]
-  }
+  # This API is still being used by the preferences page
+  sources += [
+    "api/brave_wallet_api.cc",
+    "api/brave_wallet_api.h",
+  ]
+  deps += [
+    "//brave/components/brave_wallet/browser",
+    "//brave/components/brave_wallet/common:buildflags",
+    "//components/prefs",
+  ]
 
```

Comment:
Should we also drop these remaing extension functions eventually? Worth an issue and link in comment if yes